internal/http/webserver.go

```diff
@@ -82,3 +97,41 @@ func unsuccessfulTxs(logger *zap.Logger, storage relay.Storage) http.HandlerFunc
 		}
 	}
 }
+
+func resubmitFailedTxs(logger *zap.Logger, store relay.Storage, txProcessor relay.TXProcessor, submittedTxsTasksQueue chan relay.PendingSubmittedTxInfo) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		reqBody := ResubmitRequest{}
+		decoder := json.NewDecoder(r.Body)
+		err := decoder.Decode(&reqBody)
+		if err != nil {
+			logger.Error("failed to decode request body of resubmitFailedTxs", zap.Error(err))
+			http.Error(w, fmt.Sprintf("Error processing request: %s", err), http.StatusInternalServerError)
+			return
+		}
+
+		for _, txInfo := range reqBody.Txs {
+			logger.Debug("resubmitting tx", zap.Uint64("query_id", txInfo.QueryID), zap.String("hash", txInfo.Hash))
+			tx, err := store.GetCachedTx(txInfo.QueryID, txInfo.Hash)
+			if err != nil {
+				logger.Error("failed to get unsuccessful tx", zap.Error(err))
+				httpErrorCode := http.StatusInternalServerError
+				httpErrorMessage := fmt.Sprintf("Error processing request: %s", err)
+				if errors.As(err, &leveldb.ErrNotFound) {
+					httpErrorCode = http.StatusBadRequest
+					httpErrorMessage = fmt.Sprintf("no tx found with queryID=%d and hash=%s", txInfo.QueryID, txInfo.Hash)
```

Comment:
this useful error message is swallowed by the client due to this part of the client's code:
```go
	res, err := c.client.Do(req)
if err != nil {
	return fmt.Errorf("failed to make http request: %w", err)
}

if res.StatusCode != 200 {
	return fmt.Errorf("got unexpected http response status code: %d", res.StatusCode)
}
```
example:
```
./build/neutron_query_relayer exec resubmit-tx 111 12312

Error: failed to resubmit unsuccessful tx: got unexpected http response status code: 400
Usage:
  neutron_query_relayer exec resubmit-tx <queryID> <transactionHash> [flags]

Flags:
  -h, --help   help for resubmit-tx

Global Flags:
  -u, --url string   server url (default "http://localhost:9999")
```

I suggest to add a 400 error handling to the client's side so the user can see explanation for bad requests